docs/data/charts/components/components.md

```diff
@@ -77,6 +77,16 @@ By using `invert`, the value associated with the current mouse coordinate `y` ca
 
 {{"demo": "ScaleDemo.js"}}
 
+### Series
+
+Information about series is accessible through the `useSeries` and `use*Series` hooks.
```

Comment:
```suggestion
Series information is accessible through the `useSeries` hook for all series types, and `useXxxSeries` hook for a specific series type.
```

Reply:
It was `useXxxSeries` before and I changed it to `use*Series` 😂 

Is it a  pattern in the docs to use `Xxx` to mean anything?

Reply:
No for now we nether used either `Xxx` or `*`

I find the first one better because it give a sens of the capitalization